hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metrics/HoodieMetrics.java

```diff
@@ -274,6 +274,17 @@ String getMetricsName(String action, String metric) {
     return config == null ? null : String.format("%s.%s.%s", config.getMetricReporterMetricsNamePrefix(), action, metric);
   }
 
+  public void updateClusteringFileCreationMetrics(long durationInMs) {
+    metrics.registerGauge(getMetricsName("replacecommit", "fileCreationTime"), durationInMs);
```

Comment:
Can redirect to `publishMetrics(` instead.

Reply:
Done the change to redirect the call.